samples/Databricks-AzureSQL/DatabricksNotebooks/SQL Spark Connector - Python AAD Auth.py

```diff
@@ -0,0 +1,107 @@
+# Databricks notebook source
+# DBTITLE 1,Import Python ADAL Library
+# Microsoft ADAL Library 
+# The Python ADAL library will need to be installed.
+# Example: 'pip install adal'
+
+# Documentation
+# https://github.com/AzureAD/azure-activedirectory-library-for-python
+
+# Example
+# https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/dev/sample/client_credentials_sample.py
+
+# Note, the Python ADAL library is no longer being maintained and it's documentation suggests to use MSAL. 
+# Unfortunately, it doesn't have feature parity with adal at this point and does not support service principal authentication.
+import adal
+
+# COMMAND ----------
+
+# DBTITLE 1,Configure Connection
+# Located in App Registrations from Azure Portal
+tenant_id = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+
+# Authority
+authority = "https://login.windows.net/" + tenant_id
+
+# Located in App Registrations from Azure Portal
+resource_app_id_url = "https://database.windows.net/"
+
+# In the below example, we're using a Databricks utility that facilitates acquiring secrets from 
+# a configured Key Vault.  
+
+# Service Principal Client ID - Created in App Registrations from Azure Portal
+service_principal_id = dbutils.secrets.get("principalClientId")
+
+# Service Principal Secret - Created in App Registrations from Azure Portal
+service_principal_secret = dbutils.secrets.get("principalSecret")
+
+# SQL Server URL
+url = "jdbc:sqlserver://azuresqlserver.database.windows.net"
+
+# Database Name
+database_name = "TestDatabase"
+
+# Database Table Name
+db_table = "dbo.TestTable" 
+
+# Encrypt
+encrypt = "true"
+
+# Host Name in Certificate
+host_name_in_certificate = "*.database.windows.net"
+
+# COMMAND ----------
+
+# DBTITLE 1,Authentication Options
+# SERVICE PRINCIPAL AUTHENTICATION
+#   You will need to obtain an access token.
+#   The "accessToken" option is used in the spark dataframe to indicate this 
+#   authentication modality.
+context = adal.AuthenticationContext(authority)
+token = context.acquire_token_with_client_credentials(resource_app_id_url, service_principal_id, service_principal_secret)
+
+# Set Access Token
+access_token = token["accessToken"]
+
+# ACTIVE DIRECTORY PASSWORD AUTHENTICATION
```

Comment:
Could you please add a comment to clearly qualify that this is an alternate approach (as opposed to using the SPN based approach above)? And ideally we should split this into its own cell. Sorry for the nitpick but this would improve readability, and also lessen any changes of FAQs related to this sample.